src/plugins/discover/public/application/components/no_results/no_results.tsx

```diff
@@ -174,6 +174,7 @@ export const DiscoverNoResults = ({ queryString, query, savedQuery, timeFieldNam
   // }
 
   const [savedQueries, setSavedQueries] = useState<SavedQuery[]>([]);
+  const [sampleQueries, setSampleQueries] = useState<any>([]);
```

Comment:
can we add unit tests for this file?

Reply:
UT's for saved queries are all in this PR: https://github.com/opensearch-project/OpenSearch-Dashboards/pull/8798

Reply:
Same questions as above wrt to UTs. There looks like there's new functionality, so I'm curious how we're unit testing/validating those changes or why it doesn't matter right now.

Reply:
Added a github issue to track this UT: https://github.com/opensearch-project/OpenSearch-Dashboards/issues/8853

Reply:
Follow up: In addition to tracking unit tests, saw demonstration of code paths offline